worker/worker.go

```diff
@@ -341,7 +341,7 @@ func (w *Worker) LLM(ctx context.Context, req LlmLlmPostFormdataRequestBody) (in
 	if err != nil {
 		return nil, err
 	}
-	return w.handleNonStreamingResponse(resp)
```

Comment:
Could you also apply the following commits to your branch?

- [240e16d](https://github.com/livepeer/ai-worker/pull/210/commits/240e16dabd007ce1f0c75241aecabbbbfbcf2cef)
- [b6a790d](https://github.com/livepeer/ai-worker/pull/210/commits/b6a790df7e522444cc2799240c620207612f664d)
- [241cf7a](https://github.com/livepeer/ai-worker/pull/210/commits/241cf7a50fe52b216294aceef2f1794339bb6a6c)

We're planning to move the other pipelines into their own Docker containers to make the interface more generic and the base container leaner. With that in mind, I think it would make sense to also include the LLM pipeline in this new setup. The orchestrator experience will remain unchanged once [PR #200](https://github.com/livepeer/ai-worker/pull/200) is merged.

Could you also take a look at this commit to see if it is needed?
- [9e1c48a](https://github.com/livepeer/ai-worker/pull/210/commits/9e1c48aba7e4a2e2692e35394e80dc7d84e47a72)

Lastly, please add SDK tags where you think they fit, using [this commit](https://github.com/livepeer/ai-worker/pull/210/commits/9461530046d1388494a4c9b0daf80e4c505b975e) by @ad-astra-video as an example.

Reply:
This can also be addressed in seperate pull request so approved for now 👍.

Reply:
I get the intention of the commits but I have several points of feedback on them. Usually PR feedback is left to the author to complete, and commits aren't added outside of the realm of the author. 

- [240e16d](https://github.com/livepeer/ai-worker/pull/210/commits/240e16dabd007ce1f0c75241aecabbbbfbcf2cef) -> bench.py is also in the root folder, it's also not a runtime piece of code but a dev-tool script. Either move both or neither.

- [b6a790d](https://github.com/livepeer/ai-worker/pull/210/commits/b6a790df7e522444cc2799240c620207612f664d) -> I don't really see what this accomplishes other than nuking CI ? Models can be downloaded separately regardless 

- [9e1c48a](https://github.com/livepeer/ai-worker/pull/210/commits/9e1c48aba7e4a2e2692e35394e80dc7d84e47a72) -> Probably not the most idiomatic way to do this, but fine for now. Slipped through the cracks because I mainly tested with external container. 

- https://github.com/livepeer/ai-worker/pull/210/commits/9461530046d1388494a4c9b0daf80e4c505b975e -> I made the changes, but this naming convention isn't good, Studio should not say what our naming conventions should be, I know i'm repeating myself here. 

Overall there are too many breaking changes being made in this repo that affect open PRs and there needs to be a better culture in getting PRs across the finish line and respecting the dynamics of author and reviewer.